### PR TITLE
Tests

### PR DESCRIPTION
**Pull request by hizmailovich**
## PR-Codex overview
This PR adds integration tests for the `UserController` and `PostController` classes.

### Detailed summary
- Added integration tests for `UserController` and `PostController`.
- Added `Neo4jIntegration` class to handle integration with Neo4j.
- Added `PropertyOf` class to handle dynamic property registration.
- Added tests for iterating over and retrieving users and posts.
- Added tests for creating and updating users.
- Added tests for adding and retrieving posts for a user.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -146,18 +145,6 @@
           </execution>
         </executions>
       </plugin>
-      <plugin>
```
Comment (h1alexbel) [inline pom.xml:149]: @hizmailovich what's wrong with it?

Comment (h1alexbel) [reply pom.xml:149]: @hizmailovich this plugin is more valuable than text blocks

WDYT?

Comment (h1alexbel) [reply pom.xml:149]: @hizmailovich it's time to make a decision

Comment (hizmailovich) [reply pom.xml]: Text blocks removed

**Comment by hizmailovich** [comment]
@rultor merge

**Pull request merged**